docs/migrating/index.md

```diff
@@ -61,9 +61,35 @@ BaseRoute.template_container_properties = {"full_width_row": True}
 
 ```
 
+### Navigation Links
+
+Replace regular HTML links with `NavLink` components. Instead of using click handlers and manually managing navigation, configure the path directly on the component:
```

Comment:
```suggestion
Replace regular HTML links with `NavLink` components. Instead of using click handlers and manually managing navigation, configure the path directly on the component (either in code or in the designer):
```